### DISCUSSION TITLE
Using the "Add single comment" button unexpectedly initiates a review, and does not let me use that button anymore.

### DISCUSSION THREAD

**Discussion by wircho** [community/community#7481] [Pull Requests] [answered]
**Say I am replying to an existing review comment:**
<img width="791" alt="Screen Shot 2021-11-08 at 1 33 03 PM" src="https://user-images.githubusercontent.com/8546027/140822342-417263cb-8704-4bf3-ad4b-3cadb916597d.png">

**Then despite this NOT being a review, this shows up on top:**
<img width="203" alt="Screen Shot 2021-11-08 at 1 34 04 PM" src="https://user-images.githubusercontent.com/8546027/140822124-aa13294f-1839-4498-93ae-5b197c286cb0.png">

**And now I am not even able to "Add single comment" ever again:**
<img width="315" alt="Screen Shot 2021-11-08 at 1 35 11 PM" src="https://user-images.githubusercontent.com/8546027/140822156-4eb58f77-40b1-4f8a-ac46-6f42d88afe51.png">

**Workaround: Refresh the page after adding the comment.**

**Comment by khitrenovich** [comment]
This looks like a bug to me, and it makes the "single comment" functionality hardly usable.

**Reply by wircho** [reply]
Thanks for the validation @khitrenovich! Do you know if there's a more effective way to escalate this?

**Reply by ryancdougherty** [reply]
This same thing is happening to me. It looks like it might be an intended feature, but it completely breaks the workflow that my team uses. We _never_ use the "Add Review Comment" feature, and instead always "Add single comment"s. Why would you hide your comments until they were all "done"?

**Reply by ryancdougherty** [reply]
As a workaround, as noted in the original issue above, we can refresh the page.

But, refreshing the page is a huge hassle! Github automatically collapses code and hides comments and resets the focus to the top of the page, making it very difficult to get back to the code you were reviewing.

**Reply by mathieumallet** [reply]
I have the same issue, started happening yesterday. Refreshing 'fixes' the issue, but has to be done after each comment.. Very tedious when reviewing large Pull Requests.

**Reply by Halmo** [reply]
Also having this issue. I prefer single comment as it seems more real-time. We use the slack integration to see comments so when you use single line you see each one as someone is reviewing a PR while with the review feature you get a group of comments spammed to the channel at the end. Hopefully this isn't working as intended.

**Reply by wircho** [reply]
It is unlikely that this is intended. The fact that refreshing makes it go back to normal suggests it is a bug.

**Reply by EliRobinson** [reply]
I thought interns only worked during the summer :)

Also breaks my team's workflow.

**Reply by dlanphear9** [reply]
Same here, I noticed this two days ago as well. Really slows down work flow to comment, then have to refresh after every comment. The workaround does work, but slow.

**Reply by benbp** [reply]
There are also cases where I need to reply to multiple separate discussion threads on a PR, which aren't relevant to any staged PR review comments of my own. Missing "Add single comment" also breaks this workflow.

**Comment by anna-talley** [comment]
Thanks for this feedback! I'm on the support team here at GitHub, and can confirm we have an issue open for Engineering to investigate this further. I'll keep you posted with any updates.

**Comment by anna-talley** [comment, marked as answer]
A fix for this was just deployed. Let us know in support if you have any further trouble.

**Reply by YunyiGUANG** [reply]
![image](https://user-images.githubusercontent.com/48789869/217806059-7bbec3ad-a595-4798-8e46-3e4151509176.png)

Hello, I also have the same issue. Do you know how to solve it?
